content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -448,16 +448,19 @@ A ServiceAccount for `kube-proxy` is created in the `kube-system` namespace; the
 
 #### DNS
 
-A ServiceAccount for `kube-dns` is created in the `kube-system` namespace.
+Please note that:
+1. The CoreDNS service is named `kube-dns`. This is done to prevent any interruption
+in service when the user is switching the cluster DNS from kube-dns to CoreDNS or vice-versa.
+2. In Kubernetes version 1.11 and later, CoreDNS has become the default DNS server and you have to
+invoke kubeadm with `--feature-gates=CoreDNS=false` to install kube-dns instead.
+3. In Kubernetes versions prior to 1.11, CoreDNS has to be enabled with `--feature-gates=CoreDNS=true`.
```

Comment:
Keep this parallel to the previous item, and reword thus:
"In Kubernetes version 1.10 and earlier, you must enable CoreDNS with `--feature-gates=CoreDNS=true`."